kubejs/server_scripts/gregtech/recipes.js

```diff
@@ -126,6 +126,14 @@ const registerGTCEURecipes = (event) => {
 		.itemOutput('gtceu:sticky_resin')
 		.id('tfg:pot/sticky_resin_from_conifer_pitch')
 
+	event.remove({ id: 'tfg:fluid_solidifier/latex_heating' })
```

Comment:
Is there a specific recipe from TFG that has this ID?

If so i think it'd be better to just, remove it from the source instead of removing it here then adding it again later.

Reply:
I can't remember what this is for 🤔 missed it when merging, maybe it was from back when a bunch of recipes were in TFG-Core?

Reply:
fixed